mcstatus/responses.py

```diff
@@ -101,6 +101,11 @@ def build(cls, *args, **kwargs) -> Self:
         """
         raise NotImplementedError("You can't use abstract methods.")
 
+    def as_json(self) -> dict:
+        as_dict = asdict(self)
+        as_dict["motd"] = self.motd.to_minecraft()
+        return as_dict
```

Comment:
Though only response classes (`JavaStatusResponse` and `BedrockStatusResponse`) have this method, not `JavaStatusPlayer`. But for classes without `motd` inside, it is easier to use `dataclasses.asdict` to be honest.